xcape/core.py

```diff
@@ -104,6 +104,10 @@ def calc_cape(p, t, td, ps, ts, tds, **kwargs):
         Surface Temperature in Celsius
     tds : array-like
         Surface Dew point temperature in Celsius
+    pres_lev_pos :  array-like, optional
+        location in fortran values (1: nlev) of where p <= ps.
+        When vertical_lev='model', pres_lev_pos is set to a flag = 1
+        When vertical_lev='pressure', pres_lev_pos.shape = ps.shape
```

Comment:
changed description

Reply:
It may be confusing for python users (everything 0-based) to use use fortran indexing conventions (1-based). It's easy to add 1 internally in the call to the fortran code.

Reply:
I changed it.